frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts

```diff
@@ -39,3 +39,5 @@ export const EXPAND_WAIT = 15 * MINUTE;
 export const STORAGE_CLUSTER_TAB_CNT = 15;
 export const CAPACITY_UNIT = 'TiB';
 export const CAPACITY_VALUE = '2';
+
+export const OCS_OP_NAME = 'ocs-operator';
```

Comment:
Use full forms.
```suggestion
export const OPERATOR_NAME = 'ocs-operator';
```

Reply:
already used

Reply:
I mean you can expand OP for better readability. A minor enhancement.

Reply:
done!